### PR TITLE
Ticket 8

### PR DESCRIPTION
**Pull request by nomoreaj**

### CODE REVIEW

File: src/main/jsx/app.css [hunk 1]
```diff
@@ -1 +1,3 @@
-/** My Custome CSS **/
\ No newline at end of file
+/** My Custome CSS **/
+.form-control-feedback{display:none;}
+.has-error .form-control-feedback{display:block;}
```
Comment (alexcibotari) [inline src/main/jsx/app.css]: Delete css classes

File: src/main/jsx/componenets/project-form.jsx [hunk 2]
```diff
@@ -7,19 +7,20 @@ class ProjectForm extends Component {
...
         super(props);
         this.handleChange = this.handleChange.bind(this);
-        this.save = this.save.bind(this);
+        this.validate = this.validate.bind(this);
         this.state = {
             url: '/api/projects',
             data: {name: "", key: "", description: ""},
-            edit: false
+            edit: false,
+            tempKey: ''
```
Comment (alexcibotari) [inline src/main/jsx/componenets/project-form.jsx]: change variable name from **tempKey** to **generatedKey**

File: src/main/jsx/componenets/project-form.jsx [hunk 3]
```diff
@@ -41,38 +42,72 @@ class ProjectForm extends Component {
...
-        this.setState({ data: this.state.data});
+        var val = '';
+        if (e.target.name == 'name') {
+            var arr = e.target.value.split(" ");
+            if (arr.length > 1) {
+                for (var i = 0; i < arr.length; i++) {
+                    var uppercaseletters = '';
+                    for (var j = 1; j < arr[i].length; j++) {
+                        if (arr[i].charAt(j) == arr[i].charAt(j).toUpperCase()) {
+                            console.log(arr[i].charAt(j));
```
Comment (alexcibotari) [inline src/main/jsx/componenets/project-form.jsx]: Delete logs from console

File: src/main/jsx/app.css (hunk 1 above)
Comment (nomoreaj) [inline src/main/jsx/app.css]: Done

File: src/main/jsx/componenets/project-form.jsx (hunk 2 above)
Comment (nomoreaj) [inline src/main/jsx/componenets/project-form.jsx]: the variable is depricated

File: src/main/jsx/componenets/project-form.jsx (hunk 3 above)
Comment (nomoreaj) [inline src/main/jsx/componenets/project-form.jsx]: done

**Pull request merged**